Documentation/performance/JitOptimizerTodoAssessment.md

```diff
@@ -0,0 +1,133 @@
+Optimizer Codebase Status/Investments
+=====================================
+
+There are a number of areas in the optimizer that we know we would invest in
+improving if resources were unlimited.  This document lists them and some
+thoughts about their current state and prioritization, in an effort to capture
+the thinking about them that comes up in planning discussions.
+
+
+Improved Struct Handling
+------------------------
+
+This is an area that has received recent attention, with the first-class structs
+work and the struct promotion improvements that went in for `Span<T>`.  Work here
```

Comment:
You might want to add a reference here to https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/first-class-structs.md, although I confess that it needs to be updated with current status. There are still a number of work items left to complete, many (most?) of which have associated issues.